androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivityMetadata.java

```diff
@@ -114,6 +117,22 @@
     private static final String METADATA_DISPLAY_MODE =
             "android.support.customtabs.trusted.DISPLAY_MODE";
 
+
+    /**
+     * The screen orientation to use when launching the Trusted Web Activity. Possible values are
+     * "any",
+     * "natural",
+     * "landscape",
+     * "landscape-primary",
+     * "landscape-secondary",
+     * "portrait",
+     * "portrait-primary",
+     * "portrait-secondary".
+     */
```

Comment:
Are these values coming from the web world or from the Android world? (Either way, can you add a link to where they come from?)

Reply:
This is from the w3.org spec: https://www.w3.org/TR/screen-orientation/#screenorientation-interface.

Will add this link to the new patch.